examples/plot_line_testing.py

```diff
@@ -0,0 +1,134 @@
+"""
+Test Line Plots with Matplotcheck
+=================================
+
+These are some examples of using the basic functionality of Matplotcheck
+to test line plots (including regression lines) in Python.
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# You will start by importing the required packages and plotting a line plot.
+
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from scipy import stats
+import pandas as pd
+import matplotcheck.base as pt
```

Comment:
This is not an ideal import convention `pt`. 

Nathan - let's look into being able to import ANY of the mpc tester objects using
`mpc.RasterTester`, `mpc.VectorTester`, `mpc.PlotTester`

i think we can do this by modifying the init file or similar to what they do in geopandas??